airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -102,8 +110,12 @@ export const connectionValidationSchema = (mode: ConnectionFormMode) =>
             .object({
               cronExpression: yup
                 .string()
+                .trim()
                 .required("form.empty.error")
-                .test("validCron", "form.cronExpression.error", validateCronExpression),
+                .test("validCron", "form.cronExpression.error", validateCronExpression)
+                .test("validCronFrequency", "form.cronExpression.underOneHourNotAllowed", (expression) =>
+                  allowSubOneHourCronExpressions ? true : validateCronFrequencyOneHourOrMore(expression)
```

Comment:
nit: this can be simplified to `allowSubOneHourCronExpressions || validateCronFrequencyOneHourOrMore(expression)`